search-enterprise/latest/index.md

```diff
@@ -16,7 +16,7 @@ Alfresco Search Enterprise consists of the following components:
 
 * Alfresco Content Services 7.3
```

Comment:
Will this need to be updated to ACS 7.4 later (if compatible)?

Reply:
Cheers updated.